tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -558,7 +575,7 @@ def log_call_r(x):
         return log(x)
 
     f1 = epyccel(log_call_r, language = language)
-    x = uniform(low=min_float, high=1e6)
+    x = uniform(low=0, high=max_float)
```

Comment:
For the logarithm 0 is not an acceptable argument, so you should choose a very small positive lower bound. For example abs(min_float) should work.

Reply:
Or `sys.float_info.min` ;)